cocktails.web/.env.loc

```
@@ -7,11 +7,11 @@ VITE_AUTH0_DOMAIN=login.cezzis.com
 VITE_AUTH0_CLIENT_ID=HZXyiZxjHgkQqeb6UJxmikCGYpSz5iPb
 VITE_AUTH0_REDIRECT_URI=http://localhost:4002/iam/auth/redirect/
 VITE_AUTH0_COCKTAILS_API_AUDIENCE=https://cezzis-cocktails-api
+VITE_AUTH0_ACCOUNTS_API_AUDIENCE=https://cezzis-accounts-api
+VITE_COCKTAILS_APIM_SUBSCRIPTION_KEY=383hudiudhUJK984jdus7HDY
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_COCKTAILS_APIM_SUBSCRIPTION_KEY key should go before the VITE_TELEMETRY_KEY key

(UnorderedKey)